dasar/express-js/express-mysql.md

```diff
@@ -0,0 +1,127 @@
+# Koneksi Express dan MySQL
+
+## 1. Permasalahan
+
+Sebelumnya, kita sudah bisa membuat web server yang dapat melakukan manipulasi data, dan menerima data dari body. Namun data tersebut hanya disimpan ke variable. Masalahnya, apabila server direstart, maka data yang disimpan pada variable tersebut akan hilang, karena variable disimpan pada memory sementara (RAM).
+
+## 2. Solusi
+
+Agar data yang disimpan permanen, dan tidak hilang saat server direstart, maka data tersebut harus disimpan ke dalam suatu file. File disini bisa berupa file text, atau pada kasus ini lebih cocok menggunakan database seperti MySQL.
+
+## 3. Penjelasan Detail Materi
```

Comment:
Ini kata - katanya gak perlu ngikutin di issue sih, langsung to the point aja apa yang mau dijelasin

```suggestion
## 3. Mengkoneksikan Express ke Database MySQL
```